src/main/java/com/samoilenko/onlinebookstore/dto/BookRequestDto.java

```diff
@@ -1,12 +1,18 @@
 package com.samoilenko.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class BookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+    @NotNull
```

Comment:
I could be wrong, but as far as I know, for strings it is better to use NotBlank instead of NotNull.